### ISSUE TITLE
Repo + license question

### ISSUE THREAD

**Issue by CoconutMacaroon** [tofu99/GR-2.28.2020#22]
Should this project be in its own repository? It is 100% up to you, but I think it might be helpful. Also, I realized that this doesn't have a license... and this project not having its own repo could make it difficult to add one, if you want. It is up to you, but I think that it having its own repo, and giving the project a license might be helpful. This project is just us three, so as I said earlier, it is 100% your choice, but take a look at [this](https://opensource.guide/legal/#why-do-people-care-so-much-about-the-legal-side-of-open-source). The important part of it (to me) is this: "If you don’t apply an open source license, everybody who contributes to your project also becomes an exclusive copyright holder of their work. That means nobody can use, copy, distribute, or modify their contributions – and that “nobody” includes you."... so that could be confusing. And right now, as it is just the three of us, it is probably fine to not have a license, but if you want this to be a longer-term thing, it is something @tofu99  might want to think about.

**Comment by BenDawgAK** [comment]
Probably a good idea to move the project to it's own repo.
That way we can focus the project and not screw up emery's stuff. 
I've had to clean out so many forks of this with stuff i didn't want.

**Comment by CoconutMacaroon** [comment]
About adding a license... take a look at these:
- https://opensource.stackexchange.com/questions/33/how-can-a-project-be-relicensed This talks about adding one, and that all contributors (us three) would need to give permission to add a license
- And this as well https://opensource.stackexchange.com/questions/221/can-i-license-a-project-which-i-have-already-released-without-a-license
- And [this](https://opensource.guide/legal/#why-do-people-care-so-much-about-the-legal-side-of-open-source) explains what happens if a license isn't added, and in a nutshell, it says this "If you don’t apply an open source license, **everybody who contributes to your project also becomes an exclusive copyright holder of their work. That means nobody can use, copy, distribute, or modify their contributions – and that “nobody” includes you**." (emphisis is mine)... and that is messy

**Comment by BenDawgAK** [comment]
yeah, even though this a bunch of preteens working on this, some jerk might find it, take the system, and get this shutdown

**Comment by CoconutMacaroon** [comment]
A couple of things come to mind when picking a license that @tofu99 might want to think about:
- Do we want derivative works?
- Do we want to allow commercial use?
- Or are we OK with Copyright... (instead of an open-source license)

I think we should have a license... simply because not having one is complicated ([see this](https://opensource.stackexchange.com/questions/1154/how-does-githubs-forking-right-cope-with-an-all-rights-reserved-project/1157)), but we don't have to. @tofu99 Any thoughts or ideas?

**Comment by tofu99** [comment]
I don't think we should allow commercial use, derivative works would be fine as long as they carry a different name.

**Comment by CoconutMacaroon** [comment]
Should it be Share-Alike? In other words, should they have to use the same license for derivatives? I think yes, but up to you

**Comment by BenDawgAK** [comment]
Yeah. That's usually the better, more ethical, way of doing it.

**Comment by CoconutMacaroon** [comment]
Which one? They have to have the same one or not?

**Comment by BenDawgAK** [comment]
That they have to keep the original licence

**Comment by CoconutMacaroon** [comment]
Here is what I think would need to happen
- [ ] Pick a license
- [ ] Transfer issues (if any)
- [ ] Approve all PRs (in this repo) prior to transferring it
- [ ] Move RPG to a new repo
- [ ] Add license in the new repo

**Comment by CoconutMacaroon** [comment]
Can everyone reply and say something along the lines of "I give my permission for all my contributions to this project to be re-licensed under the CC BY-NC-SA 4.0 License"? Just so that we can formally (and fairly) add a `license.md` file to the new repo with that license?

**Comment by CoconutMacaroon** [comment]
Do we still want to do this @BenDawgAK @tofu99 ? Or not?

**Comment by BenDawgAK** [comment]
yeah im cool with that
legal stuff yay

**Comment by CoconutMacaroon** [comment]
I am fine with changing the repo license.
